### DISCUSSION TITLE
All demosaic function reports "unhashable type: 'list'"

### DISCUSSION THREAD

**Discussion by BAJimH** [colour-science/colour-demosaicing#56] [Q&A]
Hi,
Thanks for this lib
I wonder why the function demosaicing_CFA_Bayer_bilinear(CFA) reports "TypeError: unhashable type: list " while I input a 2D numpy array.
I also try the example in your document, that is
`demosaiced = demosaicing.demosaicing_CFA_Bayer_bilinear(np.array(
        [
            [0.30980393, 0.36078432, 0.30588236, 0.3764706],
            [0.35686275, 0.39607844, 0.36078432, 0.40000001],
        ]))`
It reports the same error.
could you please tell me the probable cause?

**Comment by thomasmansencal** [comment]
Hi @BAJimH,

Are you running on the latest? We released a new version (0.2.4) a few hours ago that is compatible with [Colour 0.4.3](https://github.com/colour-science/colour/releases/tag/v0.4.3). I can confirm that the examples run properly with it.

Cheers,

Thomas

**Reply by BAJimH** [reply]
Hi Thomas,
Thanks for your reply very much!
When I tried to reinstall the colour-demosaicing package, I found that my version is (0.2.3). After I updated it to (0.2.4), the problem disappeared itself.

Thanks,
Jim

**Reply by thomasmansencal** [reply]
Glad to hear it is working now! :)
